VimeoUpload/UIKit/DescriptorKVObserver.swift

```diff
@@ -64,7 +64,7 @@ public class DescriptorKVObserver: NSObject
         {
             if let descriptor = self.descriptor
             {
-                let type = descriptor.type
+                let type = descriptor.descriptorType
```

Comment:
Need to make sure this isn't a breaking change. Had to rename `type` -> `descriptorType` due to compiler confusion with global namespace function `type(of:)`

Reply:
This caused no issues uploading using old or new upload. Will be tested in context of client in https://vimean.atlassian.net/browse/VIM-6844